app/views/fragments/javascriptFirstSteps.scala.html

```diff
@@ -42,11 +42,12 @@
     ) {
     guardian.stripeCheckout = {
         key: '@backend.paymentService.stripeService.publicKey',
-        image: '@hashedPathFor("images/logos/gu.png")',
+        image: 'https://d24w1tjgih0o9s.cloudfront.net/gu.png',
```

Comment:
Discussed the location with @paulbrown1982, we need to use the same icon for membership and card updates in profile.